### DISCUSSION TITLE
Congrats from old open source hand (OpenCV)

### DISCUSSION THREAD

**Discussion by garybradski** [Genesis-Embodied-AI/genesis-world#131] [General]
Just found this project, y'all are awesome (from a way back founder of OpenCV). I was just thinking that one of the key bottlenecks to embodied AI is simulation, and wishing someone would update simulators with AI prompts. Decided to search on it and here you are! Kudos!!!

I haven't even read the documents yet, but it will really boost usability if 3D scans, CAD, drawings, maps of real scenes and/or real robots can be made easy as the simulation space. Most robots are moderately bespoke -- a factory work cell, a house to clean, an office floor to polish ... even a town to deliver supplies to by carts or drones. CARLA covers this a bit, but being able to simulate actual scenes with a scan, diagram, map ... would be a super boost for training and tuning real robots. Great work!

**Comment by zhouxian** [comment]
Thank you so much for this! This is really encouraging. Yes, simulation + AI prompts is the high level scheme that we believe will unlock huge potential.
